### ISSUE TITLE
1.12.2?

### ISSUE THREAD

**Issue by NPCtom** [Pokechu22/WorldDownloader#105]
When will WDL be released for LiteLoader 1.12.2?

**Comment by Seferan** [comment]
Sep 19, 2017

**Comment by Pokechu22** [comment]
I've already supported 1.12.2 for over a year... due to me building for a ton of versions and how versions are sorted, it might be a bit hard to spot in the list now, but here's an image that should help you spot it on [the releases page](https://github.com/Pokechu22/WorldDownloader/releases):

![therelease](https://user-images.githubusercontent.com/8334194/48176079-7b4dda80-e2c3-11e8-9130-b6eaa8ebe1f4.PNG)

If you mean to ask for liteloader for 1.**13**.2, then there is no liteloader version for that since liteloader for 1.13 isn't out yet.  But 1.12.2 has been available for quite a while.

**Comment by Pokechu22** [comment]
Actually, I do see why you were confused by this with the somewhat strange sorting of releases.  I noticed in your previous issue (#104) that you were running 1.12 but didn't think to ask why (I assumed that it was compatibility with other mods).

I recently made a change to the source folders to organize them as 1.12.0, 1.12.1, 1.12.2 so that they'd sort that way instead of as 1.12.1, 1.12.2, 1.12 (commit 092948958001e5355b16d8113bb2321699e87cec); I probably should do the same thing for releases themselves since I imagine you're not the only person who has been confused by this.
